pkg/ebpf/offsetguess.go

```diff
@@ -37,6 +37,11 @@ const (
 	// The source port is much further away in the inet sock.
 	thresholdInetSock = 2000
 
+	// When GETSOCKOPT is called the TCP socket field representing rttVar (mdev_us)
+	// loses 2 bits of precision due to the operation (mdev_us >> 2) << 2, so we use
+	// this mask to correct that imprecision when comparing the values.
+	rttVarMask = ^C.__u32(3)
```

Comment:
You probably already did, but we should make sure we guarantee that this holds true for most/all kernel versions we care about :)